apps/rod/sections/summary-data-sections.js

```diff
@@ -96,16 +96,27 @@ module.exports = {
     {
       field: 'document-type',
       parse: (value, req) => {
-        const yourDocuments = Array.isArray(value) ?
-          value.map(option => option === 'Other' ? req.sessionModel.get('enter-document-type') : option).join(', ') :
-          value;
+        if (!value || (Array.isArray(value) && value.length === 0)) {
+          req.sessionModel.set('yourDocuments', 'Not provided');
+          return 'Not provided';
+        }
+        const yourDocuments = Array.isArray(value)
+          ? value.map(option =>
+            option === 'Other'
+              ? req.sessionModel.get('enter-document-type') || 'Other (not specified)'
+              : option
+          ).join(', ')
+          : value;
         req.sessionModel.set('yourDocuments', yourDocuments);
```

Comment:
We probably don't need all of these extra changes. Before setting the values in session, check whether `yourDocuments` is empty and set it to default string if it is.